docs/11.1/api/core.md

```diff
@@ -1415,16 +1415,23 @@ in the right way.
 
 _core.loadData(data, [source])_
 
-Loads new data to Handsontable. Loading new data resets the cell meta.
-Since 8.0.0 loading new data also resets states corresponding to rows and columns
-(for example, row/column sequence, column width, row height, frozen columns etc.).
+The `loadData()` method:
+- Replaces Handsontable's [`data`](@/api/options.md#data)
+- Resets cells' [configuration options](@/guides/getting-started/setting-options.md)
+- Resets rows' and columns' states (e.g. [row height](@/guides/rows/row-height.md), [column width](@/guides/columns/column-width.md), frozen [rows](@/guides/rows/row-freezing.md) and [columns](@/guides/columns/column-freezing.md) etc.)
```

Comment:
1. That's generally true when manually setting the column widths/row heights using `manualaColumnResize`/`manualRowResize`, but if they're fixed in the config object during initialization, they'll stay the same after calling `loadData`. For example:
```
const hot = new Handsontable(container, {
      colWidths: [100, 20]
});
```

or 

```
const hot = new Handsontable(container, {
      manualColumnWidths: [100, 20]
});
```

2. I don't think it's correct in terms of freezing/fixing columns/rows. They seem to remain unchanged regardless of the method being used:
- `loadData`: https://jsfiddle.net/js_ziggle/sqtx87uc/
- `updateData`: https://jsfiddle.net/js_ziggle/fjL3yusx/

Basically, `loadData`  method resets:
- cell meta (cell styling, `readOnly` states, anything stored in the cell meta object)
- index mappers (mostly the row and column order)

Reply:
@jansiegel - please have a look now at the updated version: 64e6f14